src/components/Header.jsx

```diff
@@ -68,23 +76,32 @@ const Header = ({
     }
   }
 
+  let openChat = !chatClosed ? null : (
+    <div title="Open Chat" id='open-chat-btn' onClick={showChat}>
```

Comment:
Can we have `open-chat-btn` be a CSS class instead?

Reply:
Noted